How-we-use-pull-requests.md

```diff
@@ -0,0 +1,68 @@
+## Table of contents
+
+* [PR lifecycle](#pr-lifecycle)
+* [Draft pull requests](#draft-pull-requests)
+* [Labeling issues and pull requests](#labeling-issues-and-pull-requests)
+* [Juggling multiple pull requests](#juggling-multiple-pull-requests)
+
+## PR lifecycle
+
+![Flowchart of PR lifecycle](images/prLifecycle.png)
+
+<details>
+<summary>Flowchart source code</summary>
+
+The flowchart was generated from the below source code by https://flowchart.fun.
+
+```text
+~~~
+layout:
+  name: dagre
+  rankDir: LR
+~~~
+Draft PR
+  Ready for review: PR awaiting review
+    Request changes: Changes requested
+      Addresses comments: (PR awaiting review)
+    All reviewers approve: PR labeled LGTM
+      Merge: PR closed
+```
+</details>
+
+1. (optional) Draft PR: Authors can open draft PRs to get early feedback or debug CI tests. Reviews at this stage are at the discretion of the author.
+2. PR ready for review: Once authors mark PRs as ready for review (or if they skip the draft PR stage entirely), reviews will automatically be requested from all code owners. Oppiabot will assign one developer (typically the developer from the changelog label) to take a first pass. Oppiabot will assign the other code owners to review once the first pass is complete.
```

Comment:
The PR is expected to be reviewed by the main/primary reviewer first (Currently decided by OppiaBot) and once they approve, all other code owners get assigned!

You can find more details [here](https://docs.google.com/document/d/1y2_3Ns0V00QESZy3gswc413Rki-m1exTNDt3uVcu7w0/edit#).

Reply:
Isn't that what's written? Specifically these sentences:

> Oppiabot will assign one developer (typically the developer from the changelog label) to take a first pass. Oppiabot will assign the other code owners to review once the first pass is complete.

Reply:
> reviews will automatically be requested from all code owners

Ah, I misinterpreted "requested" as assigned! (Sorry!)

> You can find more details here.

Just to confirm, this wiki page follows the process defined in the doc linked above, right?